tables/vpc_flow_log/vpc_flow_log_table.go

```diff
@@ -1,138 +1,365 @@
 package vpc_flow_log
 
 import (
-	"log/slog"
-	"time"
-
-	"github.com/rs/xid"
-
+	"github.com/turbot/go-kit/helpers"
 	"github.com/turbot/pipe-fittings/v2/utils"
 	"github.com/turbot/tailpipe-plugin-aws/sources/cloudwatch_log_group"
 	"github.com/turbot/tailpipe-plugin-aws/sources/s3_bucket"
 	"github.com/turbot/tailpipe-plugin-sdk/artifact_source"
 	"github.com/turbot/tailpipe-plugin-sdk/artifact_source_config"
+	"github.com/turbot/tailpipe-plugin-sdk/constants"
+	"github.com/turbot/tailpipe-plugin-sdk/error_types"
 	"github.com/turbot/tailpipe-plugin-sdk/formats"
 	"github.com/turbot/tailpipe-plugin-sdk/row_source"
 	"github.com/turbot/tailpipe-plugin-sdk/schema"
 	"github.com/turbot/tailpipe-plugin-sdk/table"
+	"github.com/turbot/tailpipe-plugin-sdk/types"
 )
 
 const VpcFlowLogTableIdentifier = "aws_vpc_flow_log"
-
-const vpcFlowLogFormat = `$version $account-id $interface-id $srcaddr $dstaddr $srcport $dstport $protocol $packets $bytes $start $end $action $log-status`
+const VpcFlowLogTableNilValue = "-"
+const VpcFlowLogTableSkippedData = "SKIPDATA"
+const VpcFlowLogTableNoData = "NODATA"
 
 // VpcFlowLogTable - table for VPC Flow Logs
 type VpcFlowLogTable struct {
 	table.CustomTableImpl
 }
 
+func (c *VpcFlowLogTable) Identifier() string {
+	return VpcFlowLogTableIdentifier
+}
+
 func (c *VpcFlowLogTable) GetDefaultFormat() formats.Format {
 	return defaultVPCFlowLogTableFormat
 }
 
-func (c *VpcFlowLogTable) GetSourceMetadata() ([]*table.SourceMetadata[*VpcFlowLog], error) {
+func (c *VpcFlowLogTable) GetTableDefinition() *schema.TableSchema {
+	return &schema.TableSchema{
+		Name: VpcFlowLogTableIdentifier,
+		Columns: []*schema.ColumnSchema{
```

Comment:
@ParthaI Are there any changes from the previous version of the table, or are they the same?